addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/hardware/callbacks/FritzAhaXMLCallback.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.avmfritz.internal.hardware.callbacks;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+
+import org.openhab.binding.avmfritz.internal.ahamodel.DevicelistModel;
+import org.openhab.binding.avmfritz.internal.hardware.FritzahaWebInterface;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Callback implementation for a xml response. Supports reauthorization.
+ * 
+ * @author Christoph Weitkamp
+ * 
+ */
+public class FritzAhaXMLCallback extends FritzAhaReauthCallback {

```

Comment:
Okay so your goal is to have a place to store the static final `JAXB_CONTEXT` if there is no proper location I'm okay with creating a dedicated class for it (most likely better than putting it in `FritzAhaReauthCallback ` ) but just make it a static util class and do put as a parent class to make static field accessible.

Please note that it is also not its own callback, so make it live in another package.